### DISCUSSION TITLE
Error: Windows VM does not exist - Not found in virsh list

### DISCUSSION THREAD

**Discussion by Lumipyry** [winapps-org/winapps#317] [General]
hi everyone,

my RDPWindows is in a partition mounted to `/mnt/` in `fstab` (my root partition is not big enough). I created a symlink from the RDPWindows (named win11.qrow2) to `/var/lib/libvirt/images` but WinApps install script does not find this partition:

![winapps2](https://github.com/user-attachments/assets/92059002-dcb0-4655-8280-348358e52fa4)

In my winapps.conf I have only changed username, password and WAFLAVOR="libvirt".

Is there a way to change the path WinApps searches for RDPWindows in some file (in bin/winapps for example) **OR** _is there something else wrong with my setup?_

- RDPWindows is up and running
- Symlinks from the partition in `/mnt/` work as expected to user home partition

**Comment by oskardotglobal** [comment]
I'm sorry but what does your VM being anywhere has to do with anything? Does the VM show up when running `virsh list`?

**Comment by Lumipyry** [comment]


**Reply by Lumipyry** [reply]
OK. I solved it.

This seems to be a` Debian `thing. Following instructions [here](https://serverfault.com/questions/803283/how-do-i-list-virsh-networks-without-sudo) by _Orlando Nuske_ I needed to edit a line in file `/etc/libvirt/libvirt.conf` and uncomment:
`uri_default = "qemu:///system"` and to copy the `libvirt.conf`-file to `~/.config/libvirt`.

Then I was able to see my VM in `virsh list`. However there was still something - The install gave error for FreeRDP:

![Screenshot_20241105_203430](https://github.com/user-attachments/assets/344cf371-ea93-4ff8-8962-b061cb4eb0e5)

I fixed this by removing and reinstalling both `winApps` and `winApps.src` and clearing certificate in `~/.config/freerdp/server`.

**Reply by paidlowered** [reply]
You're a lifesaver.

If anyone needs an automated way of doing this:
`echo 'uri_default = "qemu:///system"' >> ~/.config/libvirt/libvirt.conf` should work

**Comment by wamserma** [comment]
Came across this issue (or a related one) today (forgot to check which Linux distro, not my machine) being ask by someone who was stuck installing the Fmstrat fork. Issue was solved by adding `--connect qemu:///system` to the two calls of virsh. Maybe this can also help with this maintained fork (e.g. check first whether an unprivileged user can connect with/without that option and use sppropriate virsh call throughout the script).

**Reply by oskardotglobal** [reply]
We've recently addressed this problem in the documentation, see the libvirt documentation
You need to edit your `/etc/environment` to add an env var, then virsh can detect it properly.

**Comment by Lumipyry** [comment]
Similar problem and more solutions in [349](https://github.com/winapps-org/winapps/discussions/349)
